### PR TITLE
doc(fleet): openapi spec update

### PR DESCRIPTION
**Pull request by mclcavalcante**
add routes:

- /agents/backends/agents 
- /agents/backends/pktvisor/taps 
- /agents/backends/pktvisor/inputs 
- /agents/backends/pktvisor/handlers 
- /agents/backends/{backend}/

### CODE REVIEW

File: fleet/api/http/openapi.yaml [hunk 1]
```diff
@@ -250,6 +250,104 @@ paths:
...
+          $ref: "#/components/responses/AgentBackendsObjRes"
+        '400':
+          description: Failed due to malformed query parameters.
+        '401':
+          description: Missing or invalid access token provided.
+        '404':
+          description: A non-existent entity request.
+        '500':
+          $ref: "#/components/responses/ServiceErrorRes"
+  /agents/backends/{backend}/:
```
Comment (dscabral) [inline fleet/api/http/openapi.yaml:272]: It's necessary the slash on the end

**Review by dscabral — CHANGES_REQUESTED** [review]
After this, you can land this review to Shannon

File: fleet/api/http/openapi.yaml [hunk 2]
```diff
@@ -250,6 +250,82 @@ paths:
...
+          description: Missing or invalid access token provided.
+        '404':
+          description: A non-existent entity request.
+        '500':
+          $ref: "#/components/responses/ServiceErrorRes"
+  /agents/backends/pktvisor/taps:
+    parameters:
+      - $ref: "#/components/parameters/Authorization"
+    get:
+      summary: 'List of Taps available from current agents'
```
Comment (weyrick) [inline fleet/api/http/openapi.yaml:276]: let's make this description very pktvisor specific: "List of pktvisor Taps available from current agents"

File: fleet/api/http/openapi.yaml [hunk 3]
```diff
@@ -250,6 +250,82 @@ paths:
...
+          description: Missing or invalid access token provided.
+        '404':
+          description: A non-existent entity request.
+        '500':
+          $ref: "#/components/responses/ServiceErrorRes"
+  /agents/backends/pktvisor/inputs:
+    parameters:
+      - $ref: "#/components/parameters/Authorization"
+    get:
+      summary: 'List of Inputs available across current agents'
```
Comment (weyrick) [inline fleet/api/http/openapi.yaml:295]: same, "List of pktvisor Inputs..."

File: fleet/api/http/openapi.yaml [hunk 4]
```diff
@@ -250,6 +250,82 @@ paths:
...
+          description: Missing or invalid access token provided.
+        '404':
+          description: A non-existent entity request.
+        '500':
+          $ref: "#/components/responses/ServiceErrorRes"
+  /agents/backends/pktvisor/handlers:
+    parameters:
+      - $ref: "#/components/parameters/Authorization"
+    get:
+      summary: 'List of Handlers available across current agents'
```
Comment (weyrick) [inline fleet/api/http/openapi.yaml:314]: "List of pktvisor Handlers..."

**Review by weyrick — APPROVED** [review]

**Review by dscabral — APPROVED** [review]

**Pull request merged**